locale/en_US/locale.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/backup/locale/en_US/locale.xml
+  *
+  * Copyright (c) 2014-2018 Simon Fraser University
+  * Copyright (c) 2000-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
+  * Localization strings.
+  -->
+ 
+<locale name="en_US" full_name="U.S. English">
+	<message key="plugins.generic.backup.name">Backup Plugin</message>
+	<message key="plugins.generic.backup.description">This plugin generates a backup of the installation.</message>
+
+	<message key="plugins.generic.backup.link">Download Backup</message>
+	<message key="plugins.generic.backup.longdescription"><![CDATA[<p>The following links allow the Site Administrator to download a full backup of the various components of an installation. A complete backup contains <strong>all</strong> of the following components. Please refer to technical documention for more information on how these components interrelate.</p>]]></message>
+	<message key="plugins.generic.backup.db">Database</message>
+	<message key="plugins.generic.backup.files">Files</message>
+	<message key="plugins.generic.backup.code">Code</message>
+	<message key="plugins.generic.backup.db.config"><![CDATA[{$footNoteNum}. <strong>WARNING:</strong> The database dump tool has not been configured in the config.inc.php configuration file. The configuration will depend on your server configuration and the type of database you use. It should be specified in a section called [cli], in a configuration option called "dump". For MySQL, use the mysqldump tool, i.e.:<br />
+<pre>[cli]
+dump = "/usr/bin/mysqldump -h %s -u %s -p%s %s"
```

Comment:
I realize this was already implemented in the 2.x backup plugin, but I would recommend using `{$namedVariables}` rather than requiring the user to specify `%s` in a particular order.

Reply:
I am not sure what you mean here. 

At the moment this plugin reads the command from `config.inc.php` and passes it through `sprintf` (which does not support named variables).

Reply:
You'd need to change out `sprintf` for something like `str_replace`.

Reply:
These are the instructions as explained at the moment: 

```
1. WARNING: The database dump tool has not been configured in the config.inc.php configuration file. The configuration will depend on your server configuration and the type of database you use. It should be specified in a section called [cli], in a configuration option called "dump". For MySQL, use the mysqldump tool, i.e.:
[cli]
dump = "/usr/bin/mysqldump -h %s -u %s -p%s %s"
The "%s" tokens will be replaced with, in order:
Database hostname
Database username
Database password
Database name
Note that this involves specifying the database password in the command line, which may be a security concern.
```

and it's been translated to 8 languages. I totally understand your point here but that change will require updating the instructions in all the languages as well. Do you think such little gain is worth this trouble?

Reply:
In my opinion, yes. It would require removing that locale key from the languages you aren't comfortable updating yourself so that they can be re-translated. But if you feel strongly about this, it's not crucial.